titus-common-ext/elasticsearch/src/main/java/com/netflix/titus/ext/elasticsearch/model/BulkEsIndexResp.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.titus.ext.elasticsearch.model;
+
+import java.util.List;
+
+/**
+ * Elastic search data model as defined by REST API documentation
+ * https://www.elastic.co/guide/en/elasticsearch/reference/master/rest-apis.html
+ */
+public class BulkEsIndexResp {
```

Comment:
are these POJOs auto-generated code? Can they be immutable? They are inconsistent in some other ways as well, it would be nice to normalize them all with:

* many don't have private fields
* non-standard coding conventions for field and getter/setter methods on some (with underscores)
* only some implement `toString()`
* do they all need `equals()` and `hashcode()`?

Reply:
Good catch.
My initial thought on this was to not make assumptions about Webclient strategy it uses for deserializing ES responses into POJOs. But it seems like spring docs make it clear that it is going to depend on Jackson bindings for WebClient support.